<launch>
    <node pkg="camera_trigger_driver" type="gps_log_parser.py" name="gps_node" output = "screen">
    </node>
    <node pkg="camera_trigger_driver" type="camera_trigger_driver" name="camera_trigger" output = "screen">
    </node>
</launch>